app/init.php

```diff
@@ -1024,6 +1028,12 @@ function (mixed $value) {
     ]);
 }, []);
 
+//App::setResource('queue', function (Group $pools) {
```

Comment:
We probably need this now, but please avoid leaving comments in the code. If we need to trace back we can utilize Git for that.